src/components/sections/BannerText/BannerText.tsx

```diff
@@ -0,0 +1,48 @@
+import { Banner, BannerContent, BannerLink } from '@faststore/ui'
+import type { InputHTMLAttributes } from 'react'
+import React from 'react'
+
+export interface BannerTextProps extends InputHTMLAttributes<HTMLDivElement> {
+  /**
+   * The content for the h2 tag.
+   */
+  title: string
+
+  /**
+   * The content for the p tag below the h2.
+   */
+  caption: string
+
+  /**
+   * The href used at the link
+   */
+  action_path: string
```

Comment:
We use camelCase in component props

Reply:
🤣

Reply:
_Ruby feelings!_

Reply:
I got nostalgic with the launch of Rails 7